docs/book/src/quick-start.md

```diff
@@ -180,7 +180,7 @@ If you pressed `y` for Create Resource [y/n] then you created a CR for your CRD
 API definition):
 
 ```bash
-kubectl apply -k config/samples/
+make deploy
```

Comment:
that is not the same command.
make deploy will deploy the Operator itself when what is done here is apply the samples